# Service to Set the motor position
float64 angle   # Nominal position of the motor.
                # 0.0 sets the fingers perpendicular to the palm.
---
int64 ret       # Set to 1 (success) or 0 (error)
string msg      # Error description